README.md

```diff
@@ -1,62 +1,24 @@
-# Conduit Connector Template
-This is a template project for building [Conduit](https://conduit.io) connectors in Go. It makes it possible to
-start working on a Conduit connector in a matter of seconds.
+# Conduit Connector for gRPC Client
+The gRPC Client connector is one of [Conduit](https://conduit.io) plugins. It provides a destination gRPC Client connector.
 
-This template includes the following:
-* Skeleton code for the connector's configuration, source and destination
-* Example unit tests
-* A [Makefile](/Makefile) with commonly used targets
-* A GitHub workflow to [build the code and run the tests](/.github/workflows/build.yml)
-* A GitHub workflow to [run a pre-configured set of linters](/.github/workflows/lint.yml)
-* A GitHub workflow which [automatically creates a release](/.github/workflows/release.yml) once a tag is pushed
-* A [dependabot setup](/.github/dependabot.yml) which checks your dependencies for available updates and 
-[merges minor version upgrades](/.github/workflows/dependabot-auto-merge-go.yml) automatically
-* [Issue](/.github/ISSUE_TEMPLATE) and [PR templates](/.github/pull_request_template.md)
-* A [README template](/README_TEMPLATE.md)
+## How to build?
+Run `make build` to build the connector.
 
-### How to use
-* On this repository's main page, click "Use this template"
-* Enter the information about your repository
-* Once your repository has been generated, clone it
-* After cloning, run `./setup.sh <module name here>` (for example: 
-`./setup.sh github.com/awesome-org/conduit-connector-file`)
-* (Optional) Set the code owners (in the `CODEOWNERS` file)
+## Testing
+Run `make test` to run all the unit tests. Run `make test-integration` to run the integration tests.
 
-With that, you're all set up and ready to start working on your connector! As a next step, we recommend that you 
-check out the [Conduit Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk), which is the Go software 
-development kit for implementing a connector for Conduit.
+The Docker compose file at `test/docker-compose.yml` can be used to run the required resource locally.
```

Comment:
It's not being used at the moment (and it's also just the hello world image), so maybe we can remove it?